Fix community feedback (#1506)
emqtt/emq-docs-cn: en_US/getting-started/log.md

log.rotation.count = 5
```

:::tip

> When starting with the `emqx console` command, the startup script overrides `log.to` with the environment variable to set it to `console`, and the logs are not output to file.

Reviewer: ```suggestion
By default, the log files will be output as files. However, when starting with the `emqx console` command, the startup script will set `log.to = console` by environment variable and the log files will be generated to the console.
```

> By default, the log files will be output as files. However, when starting with the `emqx console` command, the startup script will set `log.to = console` by environment variable and the log files will be generated to the console.

Reviewer 2: ```suggestion
By default, the log data will be output to files. However, when starting with the `emqx console` command, the startup script will set `log.to = console` by environment variable. In that case, the log data will be output to the console.
```

log.rotation.count = 5
```

:::tip

> By default, the log data will be output to files. However, when starting with the `emqx console` command, the startup script will set `log.to = console` by environment variable. In that case, the log data will be output to the console.

Reviewer 3: ```suggestion
By default, the log data will be written to files. However, when starting node with the `emqx console` command, the startup script will apply the 'log.to = console' setting. As a result, the log data will be directed to the console instead.
```
